tutorials/scripting/c_sharp/c_sharp_signals.rst

```diff
@@ -0,0 +1,147 @@
+.. _doc_c_sharp_signals:
+
+C# Signals
+==========
+
+For a detailed explanation of signals in general, see the :ref:`doc_signals` section in the step
+by step tutorial.
+
+If it is still possible to use signals through the ``Connect``/``Disconnect`` API, C# gives us
+a more idiomatic way into the observer pattern.
```

Comment:
```suggestion
a more idiomatic way in form of the observer pattern.
```

Reply:
I'm not sure if that says the same thing? But I'm also pretty sure my sentence is not correct. The intended meaning here is "a more idiomatic way to write observer patterns".

Reply:
@paulloz "...a more idiomatic way to implement the observer pattern" then, maybe? May also useful to throw a link explaining that pattern in there.

Reply:
Sure, should I link to [this page](https://docs.godotengine.org/en/latest/getting_started/introduction/key_concepts_overview.html#signals), or directly to what the [GPP page](https://gameprogrammingpatterns.com/observer.html) linked in that note?

Reply:
@paulloz Linking to that section is fine, that piece is short enough and also means we only need to maintain that external link in one place.